graphql/queryResolvers/addStar.ts

```diff
@@ -0,0 +1,31 @@
+import db from '../../helpers/dbload'
+import { LoggedRequest } from '../../@types/helpers'
+import { Star as StarType } from '../../@types/lesson'
+
+const { Star } = db
+
+export const addStar = async (
+  _parent: void,
+  arg: StarType,
+  ctx: { req: LoggedRequest }
+) => {
+  const { req } = ctx
+  try {
+    const { studentId, mentorId, lessonId } = arg
+    if (!studentId) {
```

Comment:
you don't need `studentId` -> this creates a security flaw where I could create stars for every student.